<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>661</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>300</width>
       <height>420</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/1234.jpg);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>621</width>
       <height>511</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(151, 106, 94, 241), stop:1 rgba(255, 255, 255, 0));
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>601</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.949, y1:0.96, x2:0, y2:0, stop:0.193182 rgba(104, 41, 41, 209), stop:0.931818 rgba(104, 0, 0, 209));
border-radius:20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>User Menu</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(255,255,255,118);
color:rgba(255,255,255,230);
padding-bottom:5px</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>310</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(255,255,255,118);
color:rgba(255,255,255,230);
padding-bottom:5px</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>171</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(129, 64, 47, 241), stop:1 rgba(255, 255, 255, 0));
background-color: qlineargradient(spread:reflect, x1:0, y1:0.937, x2:0.215909, y2:0.205, stop:0.352273 rgba(104, 41, 41, 209), stop:0.863636 rgba(104, 0, 0, 209));
border-radius:20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(170, 50, 34);
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Check Flights</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(170, 50, 34);
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Check profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(170, 50, 34);
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Show Graph</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(170, 50, 34);
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Searching</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>430</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(170, 50, 34);
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Sort data</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>250</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255,118);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Departure Date:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_6">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>310</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(255,255,255,118);
color:rgba(255,255,255,230);
padding-bottom:5px</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>180</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(170, 501, 34);
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Booking</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>430</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(170, 50, 34);
color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/image/1.png);
background-image: url(:/im/1.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_8">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>160</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(255,255,255,118);
color:rgba(255,255,255,230);
padding-bottom:5px</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_9">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(255,255,255,118);
color:rgba(255,255,255,230);
padding-bottom:5px</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_10">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>210</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0);
border:none;
border-bottom:2px solid rgba(255,255,255,118);
color:rgba(255,255,255,230);
padding-bottom:5px</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>250</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255,118);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Arrival Date:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>380</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius:5px;</string>
     </property>
     <item>
      <property name="text">
       <string>Fisrt Class</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Business Class</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Economy Class</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>380</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(255,255,255,118);
font: 12pt &quot;Times New Roman&quot;;</string>
     </property>
     <property name="text">
      <string>Select the class</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <zorder>label_14</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>plainTextEdit_4</zorder>
    <zorder>plainTextEdit_5</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>label_15</zorder>
    <zorder>plainTextEdit_6</zorder>
    <zorder>pushButton_8</zorder>
    <zorder>pushButton_9</zorder>
    <zorder>label</zorder>
    <zorder>plainTextEdit_8</zorder>
    <zorder>plainTextEdit_9</zorder>
    <zorder>plainTextEdit_10</zorder>
    <zorder>label_18</zorder>
    <zorder>comboBox</zorder>
    <zorder>label_16</zorder>
    <zorder>timeEdit</zorder>
    <zorder>timeEdit_2</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="re.qrc"/>
 </resources>
 <connections/>
</ui>
